dsnp-wallet/dsnp-wallet/Source/Views/Shared/SharedImagePickerViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
```

Comment:
Can we use an object class here rather than making it a UIViewController subclass? The view controller should initialize an object which then receives delegate/callback results to return the selected image. The issue with making this a subclass is that, by using a subclass, we're implying that there is enough functionality that is being derived that would warrant using a subclass. It really is saying, "this is a highly specific type of view controller". 

In UsApp, we use `ContentPicker.swift`. You can refer to this as an example.